source/icu.net.tests/icu.net.tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net461;net6.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
we should probably keep running tests on net461 considering we publish that version to nuget.

Reply:
I found a way to make this work. net461 isn't supported on macOS, but we can selectively target different framework versions in the github actions.